api/src/test/java/org/pmiops/workbench/api/CohortsControllerTest.java

```diff
@@ -276,6 +289,17 @@ private void stubGetWorkspace(String ns, String name, String creator, WorkspaceA
     fcResponse.setWorkspace(fcWorkspace);
     fcResponse.setAccessLevel(access.toString());
     when(fireCloudService.getWorkspace(ns, name)).thenReturn(fcResponse);
+    stubGetWorkspaceAcl(ns, name, creator, access);
+  }
+
+  private void stubGetWorkspaceAcl(String ns, String name, String creator, WorkspaceAccessLevel access) {
+    WorkspaceACL workspaceAccessLevelResponse = new WorkspaceACL();
+    HashMap<String, WorkspaceAccessEntry> acl = new HashMap<>();
```

Comment:
same comments as above. It would be nice to have an immutable class for the ACL stuff with a builder method, but this is fine.